tutorial/source/gplvm.ipynb

```diff
@@ -386,45 +394,47 @@
63ybxzfuKqmPL33sbJrG1XFJSzNTxtczvj5F\nIqHgETlVKVByUKAM7mhPHweO9rBt7xFuafs9vy7xxl8faZnKnHdO5Nk39rL/SA+Hj/UxubGeFfPP\npGXaeKaMa6C3v58p4xqoT2lTm8hIoEDJQYFSuv5+Z9POA7za+TY9ff2s+u3rvLBt/0nj1SWNfqeo\ns/pPa6xj3+Eepp82lu37jnDu9IkcONLL2dMm8J7Tx7HxDwd477QJHO7p48CRHj7SMpUp4xpontDA\npLF1NNYn6XPnWG8/TePqaaxPkV5h0iY7kfIpUHJQoFSHu7Pn7WNs7TrMc2/u47ev7OFT572Tl7bt\n40fr3uTtY30ANNYnaZk2gRe27gNg6vh63jpU/j1hzCD9ta5PJpjUWMeeQ900T2igaVwD7k5dMoEZ\njKlLYsCuA0eZMbkRiDYLTps4hoNHe2ie0EAqmaC3r5/6VIKEGeMbUjSkkiQTkEgYSTOSieOPRHjd\n3dNHMmHUpxKkEgkcQh+QShgQhV0yYdQljZ4+Z2xdFI51CaO7r5+GZAIMDCOVtBCURiph1CUT9PU7\nZsc/d10yQb87STOOZ2nmcMZyIgpcC9NaxniJxMntYdYkLPqM6f9rB5KhLZHIHeK5toim+0331R/+\n05IJo98dC8tn4PMN/P9aRv0ntsnwUKDkUGqgfOOHH+U33bvAjAThi+zpH7CML3cYv7Almv0D4Dna\n4lFIFYWM0x/GSz9nLqOh5FqGmd/V/qzxssf38E/6l+BQ/ymeufzdc/4CTs8+M7BO+D/P+gKkZ5vv\nezHY8nDLqL/CBurI9RlzjG9+/KJ/6f/PgXEtet8JNWdPm6fzgeWSDgU/3n92W+Yy9EJ+0GriR+nk\nIqzA3w7D4YaLbuWiP1ow9Ig5FBooI/qkBDNbDHwfSAJ3uvtNwzGfd019Hz37x2DeH32h8ehL744R\n2ga+J9FfVtEvkkG+PH7SAAX91qtC6AzH1/34z3/o3Z2cv7ELlF4KiZw/sCeOV0rfhY6V3qqXrmfg\nl60PfNITfgEbmd+TzHn5wCJJ95n+HP3uA399ezolw3cl39+CmfPPF2QD7+fpqz+aW/jun3hF2cz+\nzY9/87O79Ix0HHRWdvyPj/Q8+u34FFHPYb3Fo2Ap5v/7pIDPniBrTaiSf2v7UNXl+vuyrPmd/HlP\nGz+5vE4LMGIDxcySwD8ClwPbgGfMbI27b6z0vD67+PZKdykicsoZyYfZzAU63P1Vdz8G3A8sibkm\nEZFRayQHynQg85aG20KbiIjEYCQHSkHMbIWZtZtZe2dnZ9zliIicskZyoGwHZma8nhHaTuDud7h7\nq7u3Njc3V604EZHRZiQHyjNAi5nNNrN6YCmwJuaaRERGrRF7lJe795rZXwOPER02fLe7D36DEBER\nGVYjNlAA3P2fgX+Ouw4RERnZm7xERKSGjKpLr5hZJ/BGiZNPBUq7BO/wUl3FUV3FUV3FOVXrere7\nD3lU06gKlHKYWXsh17KpNtVVHNVVHNVVnNFelzZ5iYhIRShQRESkIhQohbsj7gIGobqKo7qKo7qK\nM6rr0j4UERGpCK2hiIhIRShQhmBmi81ss5l1mNnKGOb/upm9ZGbPm1l7aGsyszYz2xKeJ2eMf12o\ndbOZLapwLXeb2W4zezmjrehazOzC8Jk6zOxWK/P+rYPU9Q0z2x6W2/Nm9olq1mVmM83sF2a20cw2\nmNkXQnusyytPXXEvrzFm9rSZvRDq+rvQHvfyGqyuWJdXRp9JM3vOzB4Jr+P9eXR3PQZ5EF3S5RXg\nTKAeeAGYU+UaXgemZrX9D2BlGF4J3ByG54QaG4DZofZkBWuZD1wAvFxOLcDTwDyim8k9Cnx8GOr6\nBvBfcoxblbqAM4ALwvAE4Pdh3rEurzx1xb28DBgfhuuAdaHvuJfXYHXFurwy5vdl4EfAI7Xw86g1\nlPxq9SZeS4BVYXgVcGVG+/3u3u3urwEdRJ+hItz9V0BXObWY2RnARHd/yqNv8z0Z01SyrsFUpS53\n3+Huz4bhg8Amovv1xLq88tQ1mGrV5e5+KLysCw8n/uU1WF2Dqdr33sxmAFcAd2bNP7blpUDJrxZu\n4uXAE2a23sxWhLZp7r4jDO8EpoXhOOottpbpYbgaNX7ezF4Mm8TSq/5Vr8vMZgHnE/11WzPLK6su\niHl5hc03zwO7gTZ3r4nlNUhdEP/363vAV4D+jLZYl5cCpfZd4u7nAR8HrjWz+Zlvhr8qauJQvVqq\nBbidaFPlecAO4DtxFGFm44GfAF909wOZ78W5vHLUFfvycve+8F2fQfTX87lZ78eyvAapK9blZWaf\nBHa7+/rBxoljeSlQ8ivoJl7Dyd23h+fdwE+JNmHtCquqhOfdYfQ46i22lu1heFhrdPdd4RdBP/BP\nHN/0V7W6zKyO6Jf2fe7+UGiOfXnlqqsWlleau+8DfgEspgaWV666amB5XQx8ysxeJ9oUf5mZ/ZCY\nl5cCJb9Yb+JlZuPMbEJ6GFgIvBxqWBZGWwY8HIbXAEvNrMHMZgMtRDvchlNRtYTV8QNmNi8cTXJ1\nxjQVk/6hCv4t0XKrWl2hj7uATe7+3Yy3Yl1eg9VVA8ur2cxOC8NjgcuB3xH/8spZV9zLy92vc/cZ\n7j6L6PfSk+7+GeL+eSx1b/5oeQCfIDoS5hXga1We95lER2a8AGxIzx+YAqwFtgBPAE0Z03wt1LqZ\nChxFklXPj4lW73uItrUuL6UWoJXoB/AV4H8RTrCtcF33Ai8BL4YfpjOqWRdwCdHmhheB58PjE3Ev\nrzx1xb283g88F+b/MvD1Ur/rVaor1uWVVeMfc/wor1iXl86UFxGRitAmLxERqQgFioiIVIQCRURE\nKkKBIiIiFaFAERGRilCgiIhIRShQRESkIhQoIiJSEf8f8dztHtZcYYgAAAAASUVORK5CYII=\n",
```

Comment:
dunno why the plot now has a green line?

Reply:
I fetch your branch. It does not show the green line in my system. I suspect your `matplotlib` did some magical thing here. :)